library/core/src/char/mod.rs

```diff
@@ -214,8 +214,9 @@ impl FusedIterator for EscapeUnicode {}
 
 #[stable(feature = "char_struct_display", since = "1.16.0")]
 impl fmt::Display for EscapeUnicode {
+    #[inline]
```

Comment:
There are a couple of new `#[inline]` annotations on trait methods. Were these not getting inlined somewhere? Usually generics are always inlineable.